style.css

```diff
@@ -218,265 +170,65 @@ form ul li {
   line-height: 24px;
   letter-spacing: 0.03em;
   background-color: white;
+  border-radius: 8px;
 }
 
-.head_2 {
-  padding-bottom: 30px;
+.card-details-div div {
+  margin-bottom: 15px;
```

Comment:
- [ ] Some of your cards are well adjusted and some are not, if you check well the Figma design, you will see that the image part and the text part of the project card are equal, take an example of the image below and make the necessary changes to fix this.
-  I really appreciate your work, you did well ❤️  @david30-maker.

<img width="1259" alt="2" src="https://user-images.githubusercontent.com/77843050/215269437-8bc22bd9-d0c2-40fb-be89-e52eeceb2262.png">